shared/signaling/release-notes/web.mdx

```diff
@@ -1,6 +1,9 @@
 <PlatformWrapper platform="web">
 
 This page contains information on the following releases:
+## v1.5.1
+
+v1.5.1 was released on September 1, 2022.
```

Comment:
### Behavioral changes

In previous versions, when users fail to send a channel message due to either not log in to the SDK or not join the channel, the `CHANNEL_MESSAGE_ERR_NOT_IN_CHANNEL = 5` error code is returned.

As of this version, the circumstances that trigger this error are further refined, so that users can easily identify and troubleshoot. When a failure of sending a channel message occurs,
- The `CHANNEL_MESSAGE_ERR_USER_NOT_LOGGED_IN = 102` error code is returned if users do not log in to the RTM SDK.
- The `CHANNEL_MESSAGE_ERR_NOT_IN_CHANNEL = 5` error code is returned if users log in to the SDK whereas do not join the channel.

Reply:
Missed content.

Reply:
Done

Reply:
Code formatting.
<img width="912" alt="image" src="https://user-images.githubusercontent.com/101544891/192502928-bbdbe40e-eb3a-451f-af3f-cec179dddc41.png">